client/a8c-for-agencies/components/layout/style.scss

```diff
@@ -143,6 +152,21 @@ html,
 	}
 }
 
+.a4a-layout-header-mobile {
+	align-items: center;
+	border-bottom: 1px solid var(--color-neutral-10);
+
+	.shopping-cart {
```

Comment:
I'm not sure this would be the best place to put the shopping cart styling as it is not really part of the layout component. 🤔

Reply:
Yes, that's a good point. Thanks!